packages/core/botpress/src/web/views/FlowBuilder/sidePanel/flows/index.jsx

```diff
@@ -47,6 +41,21 @@ const getInitialState = currentFlow => {
 export default class FlowsList extends Component {
   menuButtons = {}
 
+  componentWillReceiveProps(props) {
```

Comment:
This method is deprecated and soon to be removed. Is it possible to refactor using a combination of componentDidMount / componentDidUpdate?
Ref: https://reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops